Mesh_3/include/CGAL/Mesh_3/Construct_initial_points_labeled_image.h

```diff
@@ -0,0 +1,299 @@
+
+    typedef typename C3t3::Triangulation       Tr;
+    typedef typename Tr::Geom_traits           GT;
+    typedef typename GT::FT                    FT;
+    typedef typename Tr::Weighted_point        Weighted_point;
+    typedef typename Tr::Segment               Segment_3;
+    typedef typename Tr::Vertex_handle         Vertex_handle;
+    typedef typename Tr::Cell_handle           Cell_handle;
+    typedef typename GT::Vector_3              Vector_3;
+
+    const Tr& tr = c3t3.triangulation();
+
+    typename GT::Compare_weighted_squared_radius_3 cwsr =
+      tr.geom_traits().compare_weighted_squared_radius_3_object();
+    typename GT::Construct_point_3 cp =
+      tr.geom_traits().construct_point_3_object();
+    typename GT::Construct_weighted_point_3 cwp =
+      tr.geom_traits().construct_weighted_point_3_object();
+
+    const double max_v = (std::max)((std::max)(image.vx(),
+                                               image.vy()),
+                                               image.vz());
+
+    struct Seed {
+      std::size_t i, j, k;
+      std::size_t radius;
+    };
+    using Seeds = std::vector<Seed>;
+
+    Seeds seeds;
+    Mesh_3::internal::Get_point<Point_3> get_point(&image);
+    std::cout << "Searching for connected components..." << std::endl;
+    CGAL_IMAGE_IO_CASE(image.image(), search_for_connected_components_in_labeled_image(image,
+                                                     std::back_inserter(seeds),
+                                                     CGAL::Emptyset_iterator(),
+                                                     transform,
+                                                     Word()));
+    std::cout << "  " << seeds.size() << " components were found." << std::endl;
+    std::cout << "Construct initial points..." << std::endl;
+    for(const Seed seed : seeds)
+    {
+      const Point_3 seed_point = get_point(seed.i, seed.j, seed.k);
+      Cell_handle seed_cell = tr.locate(cwp(seed_point));
+
+      const Subdomain seed_label
+        = domain.is_in_domain_object()(seed_point);
+      const Subdomain seed_cell_label
+        = (   tr.dimension() < 3
+           || seed_cell == Cell_handle()
+           || tr.is_infinite(seed_cell))
+          ? Subdomain()  //seed_point is OUTSIDE_AFFINE_HULL
+          : domain.is_in_domain_object()(
+              seed_cell->weighted_circumcenter(tr.geom_traits()));
+
+      if ( seed_label != std::nullopt
+        && seed_cell_label != std::nullopt
+        && *seed_label == *seed_cell_label)
+          continue; //this means the connected component has already been initialized
+
+      const double radius = double(seed.radius + 1)* max_v;
+      CGAL::Random_points_on_sphere_3<Point_3> points_on_sphere_3(radius);
+      /// \noop [construct intersection]
+      typename MeshDomain::Construct_intersection construct_intersection =
+          domain.construct_intersection_object();
+      /// \noop [construct intersection]
```

Comment:
Nice trick with the `\noop` command, never thought about this usage.

I saw in the overnight build for the, not anymore used 1.8.13 doxygen version for the documentatioon production of CGAL 6.0 (see: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-216/logs1/Mesh_3.log), warnings like:
```
home/cgal-testsuite/cgal_doc_build/CGAL-6.0-Ic-216/include/CGAL/Mesh_3/Construct_initial_points_labeled_image.h:190: warning: Found unknown command `\noop'
```
which is correct as `\noop` is first introduced with doxygen 1.9.0 (see https://doxygen.github.io/doxygen-docs/da/de1/pg_cmds.html).
Normally doxygen only takes comments in the doxygen form (like `///`) into consideration though for the determination of the snippet start and endpoints  also normal comments (like: `//`) are taken into consideration.

As said the 1.8.13 version will not used for the documentation production anymore (of the 6.0 version of CGAL, so it is probably better not to fix this.